pkgs/tools/text/gawk/default.nix

```diff
@@ -52,6 +52,9 @@ stdenv.mkDerivation rec {
     "AR=${stdenv.cc.targetPrefix}ar"
   ];
 
+  # PIE is incompatible with the "persistent malloc" ("pma") feature.
+  hardeningDisable = [ "pie" ];
```

Comment:
That would be a full rebuild on `linux` as well. Worth adding musl-specific conditional here?

Reply:
Only if we're otherwise likely to get through staging-next without another full rebuild, in which case we could make this conditional and immediately make it unconditional in staging.  I don't like conditionals where they're not necessary, because they tend to stick around forever and cause most of the portability problems in Nixpkgs.

@vcunat how is staging-next doing? Is it worth making this conditional and then changing it back in staging to avoid a full staging next rebuild?

Reply:
I don't like full `staging-next` rebuild because of this (at this point).  Around 100k builds are finished in total.

Reply:
I'll undo the conditional in `staging` as soon as it gets merged to `staging`.

Reply:
Proposed https://github.com/NixOS/nixpkgs/pull/210258 for `staging`